sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -3170,11 +3174,23 @@ export class StorageBrowserPolicy extends BaseRequestPolicy {
     sendRequest(request: WebResource): Promise<HttpOperationResponse>;
 }
 
+// @public
+export function storageBrowserPolicy(): PipelinePolicy;
+
 // @public
 export class StorageBrowserPolicyFactory implements RequestPolicyFactory {
     create(nextPolicy: RequestPolicy, options: RequestPolicyOptions): StorageBrowserPolicy;
 }
 
+// @public
+export const storageBrowserPolicyName = "storageBrowserPolicy";
+
+// @public
+export function storageCorrectContentLengthPolicy(): PipelinePolicy;
```

Comment:
We may not expose this policy. It's just an internal implementation to correct uploading behavior, it's not for customer using.

Reply:
Since it's depended upon by other storage packages, it has to be public.  Either it has to be public here or in the common package for example.

Reply:
I see. I think we can move this to common package. I can do this later.